test/decl/protocol/special/coding/Inputs/class_codable_simple_multi1.swift

```diff
@@ -0,0 +1,25 @@
+// REQUIRES: objc_interop
```

Comment:
On Linux, objc_interop is off, but you might still have a Foundation module (swift-corelibs-foundation is pure Swift and doesn't use the ObjC runtime as you're no doubt aware).

I think tests should not pull in Foundation on Linux though, however we might still be able to test the conformance derivation by cooking up a 'mock' Foundation module that only provides the necessary protocols and nothing else.